CHANGELOG.md

```diff
@@ -22,6 +22,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 -   Remove EmailVerificationClaimClass constructor's updateContextOnIsVerifiedFalse prop
 -   Update to web-js interface version
+-   Updated supertokens website dependency, which made SessionClaimValidator a type instead of an abstract class
 
```

Comment:
does https://github.com/supertokens/supertokens-web-js/blob/build/update_website_dependency/webJsInterfaceSupported.json need to be incremented?